resources/templates/application.properties

```diff
@@ -1,2 +1,9 @@
-autoRenew.targetTypes=
-hedera.config.version=0
+ledger.id=0x01
```

Comment:
Please confirm the values of `ledger.id` and `contracts.chainId`

Reply:
Those are the values from `bootstrap.properties`. From my testing when running a version of the Network Node with the modularization enabled, only the values from `application.properties` are used.

I was running into an issue with specifically `contracts.chainId`, because it was using some other default value and not `298`, which should be used local networks

Reply:
yes, I recently came to know that `bootstrap.properties` file is no longer used.